### PR TITLE
Update device-list.md

### PR DESCRIPTION
**Pull request by rekhanr**
1. Removed HP EliteBook 830 / 840 / 850 G5 from the Active list 
2.  Moved HP EliteBook 830 / 840 / 850 G5 to the Archived list
3. Added HP EliteBook 830 / 840 / 850 G6 to the Active list
4. Added HP EliteBook x360 830 G6 Notebook PC to the Active list

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/managed-desktop/service-description/device-list.md [hunk 1]
```diff
@@ -33,8 +33,9 @@ The links to devices here are for your reference only. If you want to order devi
...
 [Dell Latitude 7200 2-in-1/7300/7400](https://www.dell.com/en-us/work/shop/dell-laptops-and-notebooks/latitude-7200-2-in-1-business-laptop/spd/latitude-12-7200-2-in-1-laptop) | 256 GB / Intel i5 / 8 GB RAM | IR camera required | May 1, 2021  | May 1, 2024  |
 [Dell Latitude 7390 7390 2-in-1 / 7490](https://www.dell.com/en-us/work/shop/cty/pdp/spd/latitude-13-7390-2-in-1-laptop?cid=265768&st=dell%2Blatitude%2B7390%2B2-in-1&VEN1=bRXisqe7,73255069985268,901q5c14135,c,,%7BProductid%7D&VEN2=be,dell%2Blatitude%2B7390%2B2-in-1&lid=5981275&dgc=st&dgseg=so&acd=12309152537461020&VEN3=112504543746142297)   | 256 GB / Intel i5 / 8 GB RAM   | IR camera required | Jan 9, 2020 | Jan 9, 2023 |
+| [HP EliteBook 830 / 840 / 850 G6 Notebook PC](https://store.hp.com/us/en/mdp/laptops/elitebook-840#!&tab=features) | 128 GB / Intel i5 / 8 GB RAM | SKU with 5VS01AV, IR Camera or fingerprint sensor required | Nov 30, 2020 | Nov 30, 2023 |
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/device-list.md]:
```suggestion
| [HP EliteBook 830 / 840 / 850 G6 Notebook PC](https://store.hp.com/us/en/mdp/laptops/elitebook-840#!&tab=features) | 128 GB / Intel i5 / 8 GB RAM | SKU with 5VS01AV, IR camera or fingerprint sensor required | Nov 30, 2020 | Nov 30, 2023 |
```

File: microsoft-365/managed-desktop/service-description/device-list.md [hunk 2]
```diff
@@ -33,8 +33,9 @@ The links to devices here are for your reference only. If you want to order devi
...
+| [HP EliteBook x360 830 G6 Notebook PC](https://store.hp.com/us/en/pdp/hp-elitebook-x360-830-g6-notebook-pc) | 128 GB / Intel i5 / 8 GB RAM | SKU with 5VS01AV, IR Camera or fingerprint sensor required | Nov 30, 2020 | Nov 30, 2023 |
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/device-list.md]:
```suggestion
| [HP EliteBook x360 830 G6 Notebook PC](https://store.hp.com/us/en/pdp/hp-elitebook-x360-830-g6-notebook-pc) | 128 GB / Intel i5 / 8 GB RAM | SKU with 5VS01AV, IR camera or fingerprint sensor required | Nov 30, 2020 | Nov 30, 2023 |
```

File: microsoft-365/managed-desktop/service-description/device-list.md [hunk 3]
```diff
@@ -58,7 +59,7 @@ These resources can help answer questions that you might have about specific dev
 
 - For more questions on Dell devices contact <a href="mailto:MMD_at_dell@dell.com">MMD_at_dell@dell.com</a>.
 
-- For more questions on HP devices contact: Americas (<a href="mailto:mmd-americas@hp.com">mmd-americas@hp.com</a>); Europe/Middle East/Africa (<a href="mailto:mmd-emea@hp.com">mmd-emea@hp.com</a>); Asia Pacific/Japan (<a href="mailto:mmd-apj@hp.com">mmd-apj@hp.com</a>); Global (<a href="mailto:mmd@hp.com">mmd@hp.com</a>)
+- For more questions on HP devices and device ordering instructions contact: Americas (<a href="mailto:mmd-americas@hp.com">mmd-americas@hp.com</a>); Europe/Middle East/Africa (<a href="mailto:mmd-emea@hp.com">mmd-emea@hp.com</a>); Asia Pacific/Japan (<a href="mailto:mmd-apj@hp.com">mmd-apj@hp.com</a>); Global (<a href="mailto:mmd@hp.com">mmd@hp.com</a>)
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/device-list.md]:
```suggestion
- For more questions on HP devices and device ordering instructions, contact: Americas (<a href="mailto:mmd-americas@hp.com">mmd-americas@hp.com</a>); Europe/Middle East/Africa (<a href="mailto:mmd-emea@hp.com">mmd-emea@hp.com</a>); Asia Pacific/Japan (<a href="mailto:mmd-apj@hp.com">mmd-apj@hp.com</a>); Global (<a href="mailto:mmd@hp.com">mmd@hp.com</a>)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by rekhanr** [comment]
@mypil , I have accepted the changes. Can you please merge the files?

**Comment by mypil** [comment]
@rekhanr - There appears to be a branch conflict to this PR. Can you please check on resolving this conflict first?

Thank you.

**Comment by mertenar** [comment]
@jaimeo can you provide assistance with this one?

**Comment by jaimeo** [comment]
I'll need to do this with @rekhanr to be sure we accept the correct version. Contact me internally....

**Pull request merged**